src/mcts/search.cc

```diff
@@ -754,9 +754,30 @@ void Search::PopulateCommonIterationStats(IterationStats* stats) {
   stats->average_depth = cum_depth_ / (total_playouts_ ? total_playouts_ : 1);
   stats->edge_n.clear();
   stats->win_found = false;
+  stats->time_usage_hint_ = IterationStats::TimeUsageHint::kNormal;
+
+  const auto draw_score = GetDrawScore(true);
+  const float fpu =
+      GetFpu(params_, root_node_, /* is_root_node */ true, draw_score);
+  float max_q = -1000;
+  uint64_t max_n = 0;
+  bool max_n_has_max_q = true;
   for (const auto& edge : root_node_->Edges()) {
-    stats->edge_n.push_back(edge.GetN());
+    const auto n = edge.GetN();
```

Comment:
Wouldn't it make sense to make this max Q+M_effect isn't max N instead of just max Q ?

Reply:
Done.